src/bundle/DependencyInjection/IbexaSystemInfoExtension.php

```diff
@@ -29,6 +29,9 @@ public function getAlias()
         return self::EXTENSION_NAME;
     }
 
+    /**
+     * @return \Ibexa\Bundle\SystemInfo\DependencyInjection\Configuration
+     */
     public function getConfiguration(array $config, ContainerBuilder $container)
```

Comment:
Return types can be narrowed down, and this class should not be extended by design (and other methods also received types).
```suggestion
    public function getConfiguration(array $config, ContainerBuilder $container): Configuration
```